php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2022,12 +2025,21 @@ class NDB_BVL_Instrument extends NDB_Page
      */
     static function loadInstanceData($instrumentInstance)
     {
-        $db       =& Database::singleton();
-        $defaults = $db->pselect(
-            "SELECT * FROM $instrumentInstance->table WHERE CommentID=:CID",
-            array('CID' => $instrumentInstance->getCommentID())
-        );
-        return $defaults[0];
+        $db = Database::singleton();
```

Comment:
This part is the main bit of the change, the rest is just so that I could still load/save after dropping the table.